Ejercicio2/Sword.java

```diff
@@ -0,0 +1,23 @@
+package Ejercicio2;
+
+public class Sword extends Weapon{
+    public String handle;
+
+    public String getHandle() {
+        return handle;
+    }
+
+    public void setHandle(String handle) {
+        this.handle = handle;
+    }
+
+    @Override
+    public String getType() {
+        return null;
+    }
+
+    @Override
+    public void setType(String type) {
+
```

Comment:
Yo need to set the attribute type. It is available because you are extending weapon